tensorboard/plugins/debugger_v2/debug_data_provider.py

```diff
@@ -177,6 +179,64 @@ def _parse_execution_data_blob_key(blob_key):
     return run, begin, end
 
 
+def graph_execution_digest_run_tag_filter(run, begin, end, trace_id=None):
```

Comment:
Are graph execution digests not keyed by both run and tag? For normal
graph summaries, at least, users use `tf.summary.trace_export(name)`,
which writes a tag name.

Reply:
In the design of DebuggerV2 backend, internally-generated tags are used to tag different types of data (e.g., see the `GRAPH_EXECUTION_DIGESTS_BLOB_TAG_PREFIX` and similar string constant tags for constructing special tags in this file.) Therefore user data is not queried by tags.